creator-node/src/routes/audiusUsers.js

```diff
@@ -14,7 +14,15 @@ module.exports = function (app) {
     const metadataJSON = req.body.metadata
 
     const metadataBuffer = Buffer.from(JSON.stringify(metadataJSON))
-    const { multihash, fileUUID } = await saveFileFromBuffer(req, metadataBuffer, 'metadata')
+    let multihash, fileUUID
+
+    try {
```

Comment:
since this adds a try/catch here i think this should be fine but let's have @SidSethi look over this to make sure nothing breaks.